### PR TITLE
Update translations

### PR DESCRIPTION
**Pull request by Bjwebb**

### CODE REVIEW

File: cove/locale/es/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -111,15 +112,25 @@ msgstr ""
...
+"Sólo podemos procesar archivos xml, csv y xlsx."
+
+#: cove/lib/exceptions.py:62
+msgid ""
+"We think you tried to supply a spreadsheet, but we failed to convert it.\n"
 "\n"
 "Error message: {}"
 msgstr ""
-"Creemos que ha tratado de suministrar una hoja de cálculo, pero no hemos podido convertirla a JSON.\n"
+"Creemos que se trató de suministrar una hoja de cálculo, pero no hemos podido convertirlo.\n"
```
Comment (edugomez) [inline cove/locale/es/LC_MESSAGES/django.po]: _pero no hemos podido **convertirlo**_ 
should be 
_pero no hemos podido convertirl**a**_

File: cove_360/locale/es/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -18,6 +18,11 @@ msgstr ""
 "Language: es\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
+#: cove_360/templates/cove_360/additional_checks.html:5
+#: cove_360/templates/cove_360/explore.html:172
+msgid "Additional Checks"
+msgstr "Comprobaciones adicionales "
```
Comment (edugomez) [inline cove_360/locale/es/LC_MESSAGES/django.po]: Trailing space

File: cove_360/locale/es/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -731,23 +729,23 @@ msgstr ""
...
 "el archivo se ha eliminado."
 
-#: cove_360/templates/cove_360/explore.html:357
+#: cove_360/templates/cove_360/explore.html:374
 msgid ""
 "These results will be available for 7 days from the day the data was first "
 "uploaded. You can revisit these results until then."
 msgstr ""
 "Estos resultados estarán disponibles durante 7 días desde el día en que los "
 "datos se cargan. Puedes revisar estos resultados hasta entonces."
```
Comment (edugomez) [inline cove_360/locale/es/LC_MESSAGES/django.po]: "_Puede**s**_" should be "_Puede_"

File: cove_360/locale/es/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -731,23 +729,23 @@ msgstr ""
...
 
-#: cove_360/views.py:36 cove_360/views.py:46
+#: cove_360/views.py:37 cove_360/views.py:47
 msgid "Sorry we can't process that data"
 msgstr "Lo sentimos, no podemos procesar esos datos"
 
-#: cove_360/views.py:38 cove_360/views.py:48
+#: cove_360/views.py:39 cove_360/views.py:49
 msgid "Try Again"
 msgstr "Inténtalo de nuevo"
```
Comment (edugomez) [inline cove_360/locale/es/LC_MESSAGES/django.po]: _"Intént**a**lo"_ should be _"Intént**e**lo"_

**Comment by Bjwebb** [comment]
@edugomez Could you update these on Transifex? https://www.transifex.com/OpenDataServices/cove/dashboard/

**Comment by edugomez** [comment]
@Bjwebb Done

**Comment by Bjwebb** [comment]
@edugomez Thanks, I've pulled those.

**Pull request merged**